_specifications/specification-3-17.md

```diff
@@ -6682,6 +6682,16 @@ export namespace CodeActionKind {
 	 */
 	export const Source: CodeActionKind = 'source';
 
+	/**
+	 * Base kind for an organize imports source action:
+	 * `source.fixAll`.
+	 *
+	 * Fix all actions automatically fix errors that have a clear fix that
+	 * do not require user input. They should not suppress errors or perform
+	 * unsafe fixes such as generating new types or classes.
```

Comment:
Some copy-paste and wording issues:

This is wrong:

> Base kind for an organize imports source action:

This sentence doesn't start well:

> Fix all actions automatically fix errors

Reply:
Eugh, good catch. The second part was copy/pasted from VS Code so I didn't want to change it too much. I've added quotes around 'Fix all' that I think make it clearer - I've filed https://github.com/microsoft/language-server-protocol/pull/1309 - comments welcome. Thanks!